src/sentry/api/endpoints/event_apple_crash_report.py

```diff
@@ -12,6 +13,7 @@
 
 @region_silo_endpoint
 class EventAppleCrashReportEndpoint(ProjectEndpoint):
+    owner = ApiOwner.OWNERS_INGEST
```

Comment:
```suggestion
    owner = ApiOwner.OWNERS_NATIVE
```

Probably, though not quite sure?